examples/mssql/failover_group/variables.tf

```
@@ -0,0 +1,11 @@
+variable "location" {
+  description = "The Azure location where the primary resources in this example should be created."
+}
+
+variable "location_alt" {
+  description = "The Azure location where the secondary resources in this example should be created."
+}
+
+variable "prefix" {
+  description = "The prefix used for all resources used by this NetApp Account"

```

this isn't a NetApp Account

```suggestion
  description = "The prefix used for all resources created as a part of this example"

```